drivers/ipfs.go

```diff
@@ -56,6 +56,10 @@ func (ostore *IpfsOS) Description() string {
 	return "Pinata cloud IPFS driver."
 }
 
+func (session *IpfsSession) IsLocationAddressable() bool {
```

Comment:
Not a regression: but we should rename this to PinataDriver/Session, and ideally not reuse the ipfs:// scheme since it we are not implementing ipfs protocol here, but pinata. I think it's safe to do even a breaking change here, I'm not aware of this being used anywhere.

Slightly more relevant now that we have another driver for an IPFS provider so the naming here makes even less sense.